Loop/Models/RemoteCommand.swift

```diff
@@ -34,6 +61,17 @@ extension RemoteCommand {
             self = .temporaryScheduleOverride(override)
         } else if let _ = notification["cancel-temporary-override"] as? String {
             self = .cancelTemporaryOverride
+        }  else if let bolusValue = notification["bolus-entry"] as? Double {
+            self = .bolusEntry(bolusValue)
+        } else if let carbsValue = notification["carbs-entry"] as? Double {
```

Comment:
Jose had this original comment on the default absorption value.  I hadn't dug in but it doesn't seem critical now either.